src/screens/transaction_details/components/msg/liquidity_provider/burn_tokens/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Trans from 'next-translate/Trans';
+import numeral from 'numeral';
+import { Typography } from '@material-ui/core';
+import { Name } from '@components';
+import { formatDenom } from '@utils/format_denom';
+import { MsgBurnTokens } from '@models';
+import { useChainContext } from '@contexts';
+
+const BurnTokens = (props: {
+  message: MsgBurnTokens;
+}) => {
+  const { findAddress } = useChainContext();
+  const { message } = props;
+
+  const liquidityProvider = findAddress(message.liquidityProvider);
+  const liqdPvdMoniker = liquidityProvider ? liquidityProvider?.moniker : message.liquidityProvider;
+
+  const amountBeforeParse = message.amount;
+  const parsedAmount = amountBeforeParse.map((x) => {
+    const eachAmount = formatDenom(x.amount, x.denom);
+    return `${numeral(eachAmount.value).format('0,0.[000000]')} ${eachAmount.denom.toUpperCase()}`;
+  });
+
+  return (
+    <Typography>
+      {
+        parsedAmount.map((x) => {
+          return (
+            <Trans
+              i18nKey="message_contents:txBurnTokens"
+              components={[
+                (
+                  <Name
+                    address={message.liquidityProvider}
+                    name={liqdPvdMoniker}
+                  />
+                ),
+                <b />,
+                <b />,
+                <b />,
+              ]}
+              values={{
+                amount: x,
+              }}
+            />
+          );
+        })
+      }
```

Comment:
What is the expected output of this i'm a bit confused

Reply:
Oh I was expecting to see all the transaction history of one person
->
Ellie burned 100NGM
Ellie burned 100EEU
Ellie burned 100STAKE
--> now I realize it should be 
Ellie burned 100NGM, 100EEUR and 100STAKE